micrometer-tracing/src/main/java/io/micrometer/tracing/SpanCustomizer.java

```diff
@@ -59,6 +74,30 @@ public SpanCustomizer event(String value) {
      */
     SpanCustomizer tag(String key, String value);
 
+    /**
+     * Sets a tag on a span.
+     * @param key tag key
+     * @param value tag value
+     * @return this, for chaining
+     */
+    SpanCustomizer tag(String key, long value);
```

Comment:
This should be `default` and delegate to the String version. Otherwise, this is a breaking change